### ISSUE TITLE
DINish is a better name than Dinish - don't change it for GF

### ISSUE THREAD

**Issue by kenmcd** [playbeing/dinish#8]
**DINish** has a link to the font history.
Users will immediately know this is a DIN style font.
**Dinish** could be anything.
Is it like Dinner?
Please do not capitulate to some arbitrary rules.
Arbitrary rules which are often broken.
GF maintains lists of the exceptions granted to this silly rule.

Abbreviations (like IBM... and DIN)
https://github.com/fonttools/fontbakery/blob/main/Lib/fontbakery/data/googlefonts/abbreviations_familyname_exceptions.txt
```
# In general, we would not like to have abbreviations
# font family names in the Google Fonts collection
# but there are a few exceptions to that rule,
# that we keep listed here, for now.

SIL
PT
IBM
STIX
VT
REM
ADLaM
AR
```
CamelCase
https://github.com/fonttools/fontbakery/blob/main/Lib/fontbakery/data/googlefonts/camelcased_familyname_exceptions.txt
```
# In general, we would not like to have camel-cased
# font family names in the Google Fonts collection
# but there are a few exceptions to that rule,
# that we keep listed here, for now.

3D # seen in "Rock 3D"
ABeeZee
BatangChe
BenchNine
BhuTuka # seen in "BhuTuka Expanded One"
BioRhyme
DotGothic # seen in "DotGothic16"
DotumChe
DynaPuff
FakeFont
GulimChe
GungsuhChe
JetBrains # seen in "JetBrains Mono"
KoHo
McLaren
MedievalSharp
MonteCarlo
MuseoModerno
RocknRoll # seen in "RocknRoll One"
UDGothic # seen in "BIZ UDGothic"
UDPGothic # seen in "BIZ UDPGothic"
UDMincho # seen in "BIZ UDMincho"
UDPMincho # seen in "BIZ UDPMincho"
UnifrakturCook
UnifrakturMaguntia
SignWriting # seen in "Noto Sans SignWriting"
WindSong
ADLaM
YouTube # seen in "YouTube Sans" 
```

So these "rules" are not in stone.
Do not bury the easily recognizable font style in the DINish name.
Dinish is meaningless and just stupid.
A newer, updated, more advanced DIN will be very popular.
Please do not lose the connection of DINish to the DIN history.
If they want the fonts, they will agree to the name.
Stick to your guns.

**Comment by driehuis** [comment]
I had already come across some of these exceptions, and privately reached the same conclusion. As a matter of fact, I've got no clue what is holding up the acceptance of DINish, seeing the fonts that got accepted in the mean time that don't meet major acceptance criteria. Mumble grumble...

Renaming the font back to DINish will hopefully also shake the maintainers of all those free fonts sites into action that still serve an ancient version of DINish that supports one fifth of the languages v3.006 sports, and misses the condensed italics... A man can dream, can't he?

So, thanks for the encouragement, your wish is my very command! I will rename it back at the next update.

**Comment by kenmcd** [comment]
Fantastic!
I have been watching the excellent progress here.
The Google Fonts approval process can be very _loooooonnnnng._
They have a relatively small team, and somewhere around a gazillion fonts in various stages (it is just crazy).
They are very knowledgeable and work very hard, but have just too much to do everyday.
So it always takes longer than font developers expect.
And not exactly sure how they prioritize.
I saw your submission in the issues tracker.
But I did not find DINish anywhere in the actual que (not even backlog).
https://github.com/orgs/google/projects/4/views/30
You are in a long line.

So just keep going like it does not matter.
One thing, they do like to have matching Italics for the other weights.
I have seen where only the Roman fonts were onboarded when there were not Italics for all the weights.
So that may be an issue in the future.

Love the font.
Glad to hear you are going to keep the historic nod to DIN in the name.

**Comment by driehuis** [comment]
Thanks for your kind words!

I will be adding the missing Bold Italic eventually. My main concern is not letting the maintenance requirements explode. There is a surprising amount of time that goes into quality control already, with the number of glyphs tripled since I started working on it. Back when I added Condensed Italic, I did not fully automate the process and that haunts me to this day. I've got most of the plumbing lying around as loose scripts and lists of rules and exceptions, so at this stage it is a [SMOP](https://en.wikipedia.org/wiki/Small_matter_of_programming).

Yeah, I'm fully aware of just how thinly stretched Google's font team is. I'm also quite aware of how far they raised the bar for quality in fonts.

Every time I hear someone fret about the quality of open source fonts I cringe. A number of times I plonked down hard cash for commercial fonts, but found production issues with them. I learned how to fix fonts from trying to use commercial fonts! Issues I reported to the vendor went unfixed. That contrasts starkly with the average font on Google Fonts, where issues are rare and tend to get quickly fixed.
